beacon-chain/db/filesystem/blob.go

```diff
@@ -179,6 +185,62 @@ func (p blobNamer) path() string {
 	return p.fname(sszExt)
 }
 
+// Prune prunes blobs in the base directory based on the retention epoch.
+// It deletes blobs older than currentEpoch - (retentionEpoch+bufferEpochs).
+// This is so that we keep a slight buffer and blobs are deleted after n+2 epochs.
+func (bs *BlobStorage) Prune(currentSlot primitives.Slot) error {
+	retentionSlot, err := slots.EpochStart(bs.retentionEpoch + bufferEpochs)
```

Comment:
Nit: Can you reconsider the naming? retentionSlot sounds like that is the slot to retain anything since thing.

```suggestion
	retentionSlots, err := slots.EpochStart(bs.retentionEpochs + bufferEpochs)
```

Reply:
Sure